packages/preview-middleware-client/src/utils/application.ts

```diff
@@ -25,3 +35,63 @@ export function getApplicationType(manifest: Manifest): ApplicationType {
 
     return 'freestyle';
 }
+
+/**
+ * Returns application object page definitions found in manifest
+ *
+ * @param manifest - manifest object
+ * @returns array with page descriptors
+ */
+export function getApplicationPages(manifest: Manifest): { id: string; entitySet: string | undefined }[] {
```

Comment:
We should keep FE related code in their separate files. We already have utility class for V4 (https://github.com/SAP/open-ux-tools/blob/main/packages/preview-middleware-client/src/utils/fe-v4.ts), we can create another one for V2.